### ISSUE TITLE
[Help]:  Read the rslc file generated by snap using pygmtsar

### ISSUE THREAD

**Issue by kjz1997** [mobigroup/gmtsar#174]
hello Dr
Hello, Doctor. I used the code in the PRM file of pygmtsar to read the rslc file generated by snap, and then I found that the read result was different from the result read by Matlab. Can you give me some suggestions? Here is the MATLAB code
`machinefmt = 'b';
fid = fopen(infile,'r',machinefmt);
[data,count]=fread(fid,bkformat);
data = complex(data(1:2:count),data(2:2:count));
 count=count/2;
width = count/lines;%
data  = reshape(data,width,lines).';`

**Comment by AlexeyPechnikov** [comment]
PyGMTSAR reads binary files in GMTSAR format. To read SNAP files, you would need to modify the code, following your MATLAB script if you believe this is correct.

**Comment by kjz1997** [comment]
OK，thank you reply,I have another problem, can pygmtsar calculate the backscatter coefficient?

**Comment by AlexeyPechnikov** [comment]
No, it has only persistent scatterers function calculation for weighted interferograms.
